src/answers-umd.js

```diff
@@ -419,8 +421,12 @@ class Answers {
   validateConfig (config) {
     // TODO (tmeyer): Extract this method into it's own class. Investigate the use of JSON schema
     // to validate these configs.
-    if (typeof config.apiKey !== 'string') {
-      throw new Error('Missing required `apiKey`. Type must be {string}');
+    if (typeof config.apiKey !== 'string' && typeof config.token !== 'string') {
+      throw new Error('Missing required `apiKey` or `token`. Type must be {string}');
+    }
+
+    if (typeof config.apiKey === 'string' && typeof config.token === 'string') {
+      throw new Error('Both apiKey and token are present. Only one authentication method should be provided');
```

Comment:
Is this really an exceptional state? I think we should ask Rose. I feel like we should just pick one and document that.

Reply:
The spec stated that if both a token and an apiKey are provided, an error should be thrown, so I think this is ok to keep